src/isPalindrome.js

```diff
@@ -1,14 +1,28 @@
-const isPalidrome = string => {
+const isPalindrome = ( string, callbackOne, callbackTwo, callbackThree ) => {
```

Comment:
This implementation is confusing to me, and does not seem to meet the specification provided in the goal?  It seems to me as if we are forced, as a user of the `isPalindrome` function, to understand *how* `isPalindrome` works, and to call it by passing in these additional callback functions...  Couldn't we just omit these from the parameters, and call them directly, instead of exporting them and forcing the user to inject them?